### PR TITLE
Updates to match `cedar` PR#260

### PR DESCRIPTION
**Pull request by khieta**
*Issue #, if available:*

*Description of changes:*

Updates to match [cedar#260](https://github.com/cedar-policy/cedar/pull/260). That PR changes the `Response` type to not be (de)serializable, but introduces a new `InterfaceResponse` type that is. This PR switches from using `cedar_policy::Response` to `cedar_policy::frontend::is_authorized::InterfaceResponse`.

Build failures are expected since the corresponding `cedar` PR is not yet merged.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cedar-drt/fuzz/fuzz_targets/rbac-authorizer.rs [hunk 1]
```diff
@@ -152,8 +153,10 @@ fuzz_target!(|input: AuthorizerInputAbstractEvaluator| {
     }
     .into();
     assert_eq!(
-        rust_res_for_comparison, definitional_res,
+        Into::<InterfaceResponse>::into(rust_res_for_comparison),
```
Comment (cdisselkoen) [inline cedar-drt/fuzz/fuzz_targets/rbac-authorizer.rs:156]:
```suggestion
        InterfaceResponse::from(rust_res_for_comparison),
```

File: cedar-drt/fuzz/src/lib.rs [hunk 2]
```diff
@@ -108,9 +109,11 @@ impl<'e> DifferentialTester<'e> {
         }
         .into();
         assert_eq!(
-            rust_res_for_comparison, definitional_res,
+            Into::<InterfaceResponse>::into(rust_res_for_comparison),
```
Comment (cdisselkoen) [inline cedar-drt/fuzz/src/lib.rs:112]:
```suggestion
            InterfaceResponse::from(rust_res_for_comparison),
```

**Review by cdisselkoen — APPROVED** [review]

File: cedar-drt/fuzz/fuzz_targets/rbac-authorizer.rs (hunk 1 above)
Comment (khieta) [reply cedar-drt/fuzz/fuzz_targets/rbac-authorizer.rs]: Thanks 🙂

**Review by shaobo-he-aws — APPROVED** [review]

**Comment by khieta** [comment]
Update: build succeeds now that the corresponding change was merged in `cedar`, and I've updated the submodule. Waiting on one final approval to merge.

**Pull request merged**